doc/base/classes.xml

```diff
@@ -5083,10 +5083,10 @@
 </class>
 <class name="Area2D" inherits="CollisionObject2D" category="Core">
 	<brief_description>
-		General purpose area detection and influence for 2D physics.
+	2D area that detects nodes that enter or exit it. Can override 2D physics properties within range.
```

Comment:
Should be one more tab.

Reply:
I thought so. Someone told me to move it back, but I'll move it up again. XD